src/main/java/com/bloomtechlabs/coderheroesbea/entities/Courses.java

```diff
@@ -0,0 +1,272 @@
+package com.bloomtechlabs.coderheroesbea.entities;
+
+
+
+import org.hibernate.annotations.Type;
+
+import javax.persistence.*;
+import java.sql.Time;
+import java.util.Date;
+import java.util.List;
+
+@Entity
+@Table(name = "courses")
+
+//@Typ
```

Comment:
Consider removing these dead code comments above the class definition

Reply:
Remove dead code comments